src/System.Runtime.Numerics/tests/BigInteger/Comparison.cs

```diff
@@ -363,11 +363,11 @@ public static void IComparable_Invalid(string paramName)
         {
             IComparable comparable = new BigInteger();
             Assert.Equal(1, comparable.CompareTo(null));
-            Assert.Throws<ArgumentException>(paramName, () => comparable.CompareTo(0)); // Obj is not a BigInteger
+            AssertExtensions.Throws<ArgumentException>(paramName, () => comparable.CompareTo(0)); // Obj is not a BigInteger
         }
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp | TargetFrameworkMonikers.Uap)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp | TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot)] // Missing Exception.ParamName fixed in .NETCore
```

Comment:
Actually, it did get fixed in corert (devexp mode). So that version runs the non "_net64" version.

(Which of course will muzzle the ParamName test anyway as we do all the others, but regardless, it doesn't make much sense to run a "_net64" test on a non-4.6 framework.)